### DISCUSSION TITLE
How to uninstall and update ytfzf? (Linux Beginner)

### DISCUSSION THREAD

**Discussion by ttx99** [pystardust/ytfzf#624] [Q&A] [answered]
I cant seem to find in the documentation.

**Comment by Euro20179** [comment, marked as answer]
If you installed it manually, you can download it again and do `make uninstall`, to update just install it again.
If you didn't install it manually I cannot help you.

**Reply by ttx99** [reply]
git clone https://github.com/pystardust/ytfzf
cd ytfzf
sudo make install doc

I entered those command still on 2.5.3

**Reply by Euro20179** [reply]
You have to run `sudo make uninstall` first.

**Reply by ttx99** [reply]
![Screenshot_2023-01-22_15-50-18](https://user-images.githubusercontent.com/95531944/213905915-941da271-14d2-4359-ab18-0948d595a0e2.png)

**Reply by Euro20179** [reply]
You need to be in the ytfzf folder.
Make is not a package manager, it looks for a file called `Makefile` and uses that.

**Reply by ttx99** [reply]
![Screenshot_2023-01-23_09-17-29](https://user-images.githubusercontent.com/95531944/213951570-6377168e-ce3d-42af-90f9-aa94665f508b.png)

**Reply by Euro20179** [reply]
you can't clone from inside the cloned directory.

You either have to fetch origin `git fetch origin` or you have to delete the folder and re clone it in order to get the updated version.

**Reply by ttx99** [reply]
done updating but this error..
![Screenshot_2023-01-23_18-53-17](https://user-images.githubusercontent.com/95531944/214022469-4a4378a8-5f3b-49b9-bfb8-aa1daaf9646c.png)

**Reply by Euro20179** [reply]
`ytfzf --rii`

**Reply by ttx99** [reply]
![Screenshot_2023-01-24_21-47-07](https://user-images.githubusercontent.com/95531944/214311639-ff51b3e1-09a7-417e-9b89-083e2fd56a73.png)

**Reply by Euro20179** [reply]
That's odd, the option did literally nothing, well you can always create that file yourself, and just put a bunch of invidious instances in it each on their own line.

**Reply by ttx99** [reply]
can you link to the default contents of the file so i can just paste it.

**Reply by Euro20179** [reply]
This is mine:
```
https://iv.ggtyler.dev
https://invidious.silur.me
https://invidious.namazso.eu
https://invidious.nerdvpn.de
https://y.com.sb
https://yt.oelrichsgarcia.de
https://vid.priv.au
https://inv.riverside.rocks
https://invidious.lidarshield.cloud
https://invidious.tiekoetter.com
https://invidious.baczek.me
https://invidious.dhusch.de
https://yt.funami.tech
https://invidious.sethforprivacy.com
https://vid.puffyan.us
https://inv.vern.cc
https://invidious.esmailelbob.xyz
https://invidious.privacydev.net
https://iv.melmac.space
https://invidious.epicsite.xyz
https://inv.bp.projectsegfau.lt
https://yt.artemislena.eu
https://invidious.flokinet.to
https://invidious.drivet.xyz
https://inv.odyssey346.dev
https://watch.thekitty.zone
```
